src/model.rs

```diff
@@ -384,6 +385,33 @@ impl Model {
                     });
                 }
 
+                Event::EvActAppendAndSelect => {
+                    let query = self.query.get_query();
+                    let tx = Arc::new(SpinLock::new(self.tx.clone()));
+                    if query.len() == 0 {
+                        let _ = tx.lock().send((Event::EvHeartBeat, Box::new(true)));
+                    } else {
+                        let mut v = Vec::new();
+                        let item = Item::new(
+                            String::from_utf8_lossy(query.to_string().as_bytes()),
+                            false,
+                            &Vec::new(),
+                            &Vec::new(),
+                            &Regex::new("").unwrap(),
+                            (std::usize::MAX, append_idx),
+                        );
+                        append_idx += 1;
+                        let item = Arc::new(item);
+                        v.push(item.clone());
+                        self.item_pool.append(v);
```

Comment:
Currently, In interactive mode, item_pool will be cleared on cmd_query change. But of course that won't be a good use case of `append-and-select` and could be ignored right now.